src/app/(tabs)/home/styles.ts

```diff
@@ -22,6 +22,7 @@ const styles = StyleSheet.create({
   scrollView: {
     marginBottom: 20,
```

Comment:
Should be `paddingBottom: 16`. Switching to padding fixes the weird line thing that you can see above "New Stories". Make sure to keep the `padding: 8` below, this should just change the bottom padding while leaving the other direction at 8.

Reply:
Changing the padding here fixes the weird line problem, but the spacing for the "Recommended for You" and "New Stories" need to be different now because the padding pushes down "New Stories" and not "Recommended for You".